node/executor/src/lib.rs

```diff
@@ -81,14 +81,14 @@ mod tests {
 			Some((signed, index)) => {
 				let era = Era::mortal(256, 0);
 				let payload = (index.into(), xt.function, era, GENESIS_HASH);
-				let key = AccountKeyring::from_public(&signed).unwrap();
-				let signature = payload.using_encoded(|b| {
+				let key = AccountKeyring::from_h256_public(signed.clone().into()).unwrap();
```

Comment:
converting via `H256` kills semantics. this stuff should retain strong typing as before.